shared/string.js

```diff
@@ -0,0 +1,27 @@
+String.prototype.isEmpty = function() {
+  return this.trim() === '';
+};
+
+String.prototype.isBlankOrGitOrBit = function() {
+  return /^$|github.com|bitbucket.org/g.test(this);
+};
+
+String.prototype.toLowerHyphenated = function() {
```

Comment:
I think the function should be only `hyphenated` since lowercase is already exist as a function.